File: GreenScreen/app/src/main/java/com/example/greenscreen/vehicles/Car.java
package com.example.greenscreen.vehicles;

import android.os.Bundle;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

import com.example.greenscreen.R;

public class Car extends AppCompatActivity {
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_car);
        TextView CO2 = (TextView) findViewById(R.id.text_view);
        CO2.setText("CO2 emissions: ");
    }
}

File: GreenScreen/app/src/main/java/com/example/greenscreen/vehicles/Car.kt
package com.example.greenscreen.vehicles

import android.os.Bundle
import android.view.View
import android.widget.EditText
import androidx.appcompat.app.AppCompatActivity
import com.example.greenscreen.R
import kotlinx.android.synthetic.main.activity_car.*
import kotlinx.android.synthetic.main.activity_walk.*
import kotlinx.android.synthetic.main.activity_walk.text_view


class Car : AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_car)
        text_view.setText("CO2 emissions: ")

       // val num1 = CO2.text.toString().toInt()
    }
}